packages/now-routing-utils/src/superstatic.ts

```diff
@@ -75,7 +75,7 @@ export function convertHeaders(headers: NowHeader[]): Route[] {
       obj[kv.key] = kv.value;
     });
     const route: Route = {
-      src: h.source,
+      src: sourceToRegex(h.source).src,
       headers: obj,
       continue: true,
     };
```

Comment:
We need to validate headers in `index.ts`.

Search for:

```
Redirect `source` contains invalid regex.
```

We need a similar validation for `headers` too.

Reply:
Added a couple tests for this